x-pack/test/api_integration/apis/management/rollup/lib/es_index.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { getRandomString } from './random';
+
+/**
+ * Helpers to create and delete indices on the Elasticsearch instance
+ * during our tests.
+ * @param {ElasticsearchClient} es The Elasticsearch client instance
+ */
+export const initElasticsearchIndicesHelpers = (es) => {
```

Comment:
Since these helpers mutate and depend upon state, why not use a class here? I expect standalone functions to be unrelated and be side-effect-free, but in this case it's the opposite, and I think a class would communicate that.

Reply:
This is how it has been done in CCR. Like always code can be written in multiple ways. I don't think it would add much if the same code would be written in a Class to be honest, as there isn't really much happening in the file.